### PR TITLE
docs(unity-bootstrap-theme): align old stories with the new figma design for the hero component

### PR DESCRIPTION
**Pull request by maycomayco**
### Description

We aligned the existing stories in the storybook with the new design for the hero component

### Links

- [Unity reference site](https://unity.web.asu.edu/)
- [JIRA ticket](https://asudev.jira.com/browse/UDS-1304)
- [Unity Design Kit OLD](https://xd.adobe.com/view/56f6cb78-9af5-4b12-b4ce-ef319f71113f-03a5/screen/2507e745-04a5-495a-a49a-f8f6bd82c155)
- [Unity Design Kit NEW](https://xd.adobe.com/view/78cbcd93-ef3e-4153-a7c5-428baae21ed1-2d98/)

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples
- [x] No new console errors
- [x] Accessibility checks

### Browsers

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] Edge

### Images
image

### CODE REVIEW

**Review by davidornelas11 — DISMISSED** [review]
Looks good!

File: packages/unity-bootstrap-theme/stories/molecules/heroes/heroes.examples.stories.js [hunk 1]
```diff
@@ -23,99 +23,134 @@ export const HeroMedium = createStory(
       fetchpriority="high"
     />
     <h1>
-      <span className="highlight-white">
-        Lorem ipsum dolor sit amet, consectetur adip
+      <span className="highlight-gold">
+        ASU named to prestigious university association
```
Comment (mlsamuelson) [inline packages/unity-bootstrap-theme/stories/molecules/heroes/heroes.examples.stories.js]: We need less words in this hero for the example. 3-4 words would be ideal - maybe "ASU named prestigious university".

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
@maycomayco Comparing the stories with the XD, I don't see a `Story Hero Medium` story  - were you considering that outside the scope? If so, can you create a separate ticket so we have an example for that, please?

**Comment by maycomayco** [comment]
> @maycomayco Comparing the stories with the XD, I don't see a `Story Hero Medium` story - were you considering that outside the scope? If so, can you create a separate ticket so we have an example for that, please?

The story for the Story Hero Medium component was defined on the examples page, now I moved that code to the templates page to keep it unified there.

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**